### PR TITLE
added vision sdk and cli pipeline job examples

### PR DESCRIPTION
**Pull request by BhavanaTumma**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+    "        target_column_name=\"label\",\n",
+    "        primary_metric=\"Accuracy\",\n",
+    "        properties={\"_automl_internal_label\": \"latest\",},\n",
+    "        # currently need to specify outputs \"mlflow_model\" explictly to reference it in following nodes \n",
+    "        outputs={\"best_model\": Output(type=\"mlflow_model\")},\n",
+    "    )\n",
+    "    \n",
+    "    image_classification_node.set_limits(timeout_minutes=180)\n",
+    "    \n",
+    "    image_classification_node.extend_search_space(\n",
```
Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:318]: Can we use the same search space as the one used in cli examples and standalone examples.

Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:318]: Can we use the same search space as the one used in cli examples and standalone examples.

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+    "        sampling_algorithm=\"Random\",\n",
+    "        early_termination=BanditPolicy(evaluation_interval=2, slack_factor=0.2, delay_evaluation=6),\n",
+    "    )\n",
+    "\n",
+    "    command_func = command(\n",
+    "        inputs=dict(\n",
+    "            automl_output=Input(type=\"mlflow_model\")\n",
+    "        ),\n",
+    "        command=\"ls ${{inputs.automl_output}}\",\n",
+    "        environment=\"AzureML-sklearn-0.24-ubuntu18.04-py37-cpu:1\"\n",
```
Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:342]: Why is this specific image used ? Does this command run inference using the mlflow model ?

Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:342]: Why is this specific image used ? Does this command run inference using the mlflow model ?

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "\n",
+    "from azure.ai.ml import MLClient, Input, command, Output\n",
+    "from azure.ai.ml.automl import image_classification\n",
+    "from azure.ai.ml.dsl import pipeline\n",
+    "from azure.ai.ml.entities._job.automl.image import ImageClassificationSearchSpace\n",
```
Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:45]: Can we import from azure.ai.ml.automl

Comment (PhaniShekhar) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:45]: Same in other notebooks as well

Comment (PhaniShekhar) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:45]: Same in other notebooks as well

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+    "                learning_rate=Uniform(0.001, 0.01),\n",
+    "            ),\n",
+    "            ImageClassificationSearchSpace(\n",
+    "                model_name=Choice([\"seresnext\", \"resnet50\"]),\n",
+    "                learning_rate=Uniform(0.001, 0.01),\n",
+    "            ),\n",
+    "        ]\n",
+    "    )\n",
+    "    image_classification_node.set_sweep(\n",
+    "        max_trials=20,\n",
```
Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:331]: Can we use the same settings used in cli examples.

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline.ipynb [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+    "        primary_metric=\"iou\",\n",
+    "        properties={\"_automl_internal_label\": \"latest\",},\n",
+    "        # currently need to specify outputs \"mlflow_model\" explictly to reference it in following nodes \n",
+    "        outputs={\"best_model\": Output(type=\"mlflow_model\")},\n",
+    "    )\n",
+    "    image_classification_multilabel_node.set_limits(timeout_minutes=120)\n",
+    "\n",
+    "    image_classification_multilabel_node.extend_search_space(\n",
+    "        [\n",
+    "            ImageClassificationSearchSpace(\n",
```
Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline.ipynb:322]: Please use the same search space as used in cli examples

Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline.ipynb:322]: Please use the same search space as used in cli examples

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-object-detection-in-pipeline.ipynb [hunk 6]
```diff
@@ -0,0 +1,449 @@
...
+    "            ImageObjectDetectionSearchSpace(\n",
+    "                model_name=Choice([\"fasterrcnn_resnet50_fpn\"]),\n",
+    "                learning_rate=Uniform(0.0001, 0.001),\n",
+    "                optimizer=Choice([\"sgd\", \"adam\", \"adamw\"]),\n",
+    "                min_size=Choice([600, 800]),  # model-specific\n",
+    "            ),\n",
+    "        ]\n",
+    "    )\n",
+    "    \n",
+    "    image_object_detection_node.set_image_model(nms_iou_threshold=0.7)\n",
```
Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-object-detection-in-pipeline.ipynb:358]: Any reason for using this parameter in the image model ? If possible, please use the same settings as in cli examples.

Comment (PhaniShekhar) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-object-detection-in-pipeline.ipynb:358]: Any reason for using this parameter in the image model ? If possible, please use the same settings as in cli examples.

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb [hunk 7]
```diff
@@ -0,0 +1,420 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import urllib\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "# download data\n",
+    "download_url = \"https://cvbp-secondary.z19.web.core.windows.net/datasets/object_detection/odFridgeObjects.zip\"\n",
```
[the remark this answers is not in the document]
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb]: Done

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb (hunk 1 above)
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:317]: Done

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb (hunk 2 above)
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb:344]: It's a placeholder for now for all pipeline examples, and will be modified later on.

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb (hunk 3 above)
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb]: Done

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb (hunk 4 above)
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb]: Done

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline.ipynb (hunk 5 above)
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline.ipynb:321]: Done

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-object-detection-in-pipeline.ipynb (hunk 6 above)
Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-object-detection-in-pipeline.ipynb]: Done

**Comment by sharma-riti** [comment]
nit: Duplicate line     image_instance_segmentation_node.set_image_model(nms_iou_threshold=0.7)
under sdk\jobs\pipelines\1h_automl_in_pipeline\automl-image-instance-segmentation-in-pipeline.ipynb

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-instance-segmentation-in-pipeline.ipynb [hunk 8]
```diff
@@ -0,0 +1,368 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get a handle to workspace\n",
+    "ml_client = MLClient.from_config(credential=credential)\n",
```
Comment (sharma-riti) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-instance-segmentation-in-pipeline.ipynb:91]: Follow the same pattern as standalone jobs for getting workspace

credential = DefaultAzureCredential()
ml_client = None
try:
    ml_client = MLClient.from_config(credential)
except Exception as ex:
    print(ex)
    # Enter details of your AML workspace
    subscription_id = "<SUBSCRIPTION_ID>"
    resource_group = "<RESOURCE_GROUP>"
    workspace = "<AML_WORKSPACE_NAME>"
    ml_client = MLClient(credential, subscription_id, resource_group, workspace)

Comment (sharma-riti) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-instance-segmentation-in-pipeline.ipynb:91]: for all jobs*

Comment (BhavanaTumma) [reply sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-instance-segmentation-in-pipeline.ipynb]: Done

File: cli/jobs/pipelines/automl/forecasting-energy-demand-pipeline/How to Run this AutoML Job with CLI (Forecasting).txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+How to Run this AutoML Job with CLI (Time-Series Forecasting)
+===============================================================
```
Comment (skasturi) [inline cli/jobs/pipelines/automl/forecasting-energy-demand-pipeline/How to Run this AutoML Job with CLI (Forecasting).txt:2]: Isn't this supposed to be an MD file?

**Review by CESARDELATORRE — APPROVED** [review]
LGTM. With that approach the examples are self-contained. :)

**Review by sharma-riti — APPROVED** [review]

File: cli/jobs/pipelines/automl/forecasting-energy-demand-pipeline/How to Run this AutoML Job with CLI (Forecasting).txt (hunk 9 above)
Comment (BhavanaTumma) [reply cli/jobs/pipelines/automl/forecasting-energy-demand-pipeline/How to Run this AutoML Job with CLI (Forecasting).txt:2]: These seem to be txt files throughout

**Pull request merged**